packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.constants.js

```diff
@@ -0,0 +1,27 @@
+export const SHARED_CDN_SETTINGS_RULE_TYPE_DEVMODE = 'devmode';
+export const SHARED_CDN_SETTINGS_RULE_TYPE_BROTLI = 'brotli';
+export const SHARED_CDN_SETTINGS_RULE_CACHE_RULE = 'cache_rule';
+export const SHARED_CDN_SETTINGS_RULE_CACHE_RULE_PATTERN_REGEX = 'regex';
+export const SHARED_CDN_SETTINGS_RULE_CACHE_RULE_PATTERN_EXTENSION =
+  'extension';
+export const SHARED_CDN_SETTINGS_RULE_FACTOR_DAY = 86400;
+export const SHARED_CDN_SETTINGS_RULE_FACTOR_HOUR = 3600;
+export const SHARED_CDN_SETTINGS_RULE_FACTOR_MINUTE = 60;
+
+export const SHARED_CDN_GET_MORE_INFO = {
+  FR: 'https://www.ovh.com/fr/hebergement-web/cdn.xml',
+  DE: 'https://www.ovh.de/hosting/cdn.xml',
+  ES: 'https://www.ovh.es/hosting/cdn.xml',
+  IT: 'https://www.ovh.it/hosting-web/cdn.xml',
+  NL: 'https://www.ovh.nl/shared-hosting/cdn.xml',
+  PL: 'https://www.ovh.pl/hosting/cdn.xml',
+  PT: 'https://www.ovh.pt/alojamento-partilhado/cdn.xml',
+  UK: 'https://www.ovh.co.uk/web-hosting/cdn.xml',
```

Comment:
```suggestion
  GB: 'https://www.ovh.co.uk/web-hosting/cdn.xml',
```

Reply:
AS reported by @marie-j & according to the `/me` API `ovhSubsidiary` valid values are `CZ`, `DE`, `ES`, `EU`, `FI`, `FR`, `GB`, `IE`, `IT`, `LT`, `MA`, `NL`, `PL`, `PT`, `SN`, `TN`